bitnami/cilium/templates/agent/rbac.yaml

```diff
@@ -0,0 +1,203 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "cilium.agent.fullname" . }}
```

Comment:
For cluster-scoped resources we use the `fullname.namespace` variant so we don't find issues when deploying in different namespaces (not sure if you may need to deploy two instances for operations like upgrades or similar as in solutions like Istio)

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/129afd439cbffb07203c84e7db2880f82caa370f